pricegraph/src/lib.rs

```diff
@@ -32,7 +32,7 @@ impl TransitiveOrderbook {
     pub fn ask_prices(&self) -> impl DoubleEndedIterator<Item = (f64, f64)> + '_ {
         self.asks
             .iter()
-            .map(|order| ((order.buy / order.sell) * FEE_FACTOR, order.sell))
+            .map(|order| ((order.sell / order.buy) / FEE_FACTOR, order.buy))
```

Comment:
Trying to find the issue (cf #1037). Is this change correct? Asks sell quote and buy base. E.g. in ETH-USD they sell USD for ETH. The price (in quote) should be around $200, but if I sell 1ETH for $200 this evaluates to 0.005 (which is backwards).

Same for bids.